packages/sdk/src/router/v6/types.ts

```diff
@@ -20,6 +21,13 @@ export type NFTApproval = {
   txData: TxData;
 };
 
+export type TokenApproval = {
```

Comment:
Since this is exactly the same with `NFTApproval` let's just rename `NFTApproval` to `Approval` and re-use it for ERC20 tokens as well.